### ISSUE TITLE
Искажение фильтров изображения и некорректное отображение луны/солнца

### ISSUE THREAD

**Issue by Daler-Karimov** [skyrim-multiplayer/issue-tracker#41]
## Описание

Появляется розовый фильтр изображения и квадрат вместо луны или солнца (см.скриншоты)

## Воспроизведение

Зайти в здание, выйти из игры, зайти в игру, выйти из здания.

## Локация, FormID

Не имеет значения.

## Вероятность воспроизведения

100%

## Ожидаемый результат

Не должно искажаться изображение

## Среда исполнения

Windows 10х64
Steam-версия Skyrim SE и пиратская копия игры.

## Видео/скриншоты

![The_Elder_Scrolls_V_Skyrim_Special_Edition_Screenshot_2020 11 10_-_21 00 52 44](https://user-images.githubusercontent.com/61763100/99003702-91946e00-254f-11eb-827d-206762cd6f39.png)
![ScreenShot89](https://user-images.githubusercontent.com/61763100/99003708-935e3180-254f-11eb-8938-d1a23d179498.png)

![ScreenShot87](https://user-images.githubusercontent.com/61763100/99004084-bbe62b80-254f-11eb-8450-ae4f132b1fbd.png)
![The_Elder_Scrolls_V_Skyrim_Special_Edition_Screenshot_2020 11 11_-_22 03 16 82](https://user-images.githubusercontent.com/61763100/99004319-eb953380-254f-11eb-8cb8-b0f745df9b8f.png)

**Comment by Pospelove** [comment]
Зашёл в 3:19 по МСК в Дом Алвора и Сигрид в Ривервуде. Вышел из игры. Вошёл в игру. Вышел из дома. Небо выглядит нормально. Не удалось воспроизвести. Прошу обновить инструкцию по воспроизведению или изменить вероятность.

**Comment by Daler-Karimov** [comment]
Дополнение 1: у других людей, при совершении действии указанных выше, начинает пропадать или мерцать луна
1-ый тестер: https://youtu.be/71w39MhKw_U

2-ой тестер: https://drive.google.com/file/d/16kxvcuRrUULkhUI_R2r9TYE2JitfuNUY/view

**Comment by Pospelove** [comment]
@Daler-Karimov Ещё проверьте, не мигают ли в это время источники света на земле (освещение в городах). Кажется, было такое

**Comment by Daler-Karimov** [comment]
@Pospelove Да, при перезаходе в игру, будучи внутри интерьерной локации типа: "Дом", сбивается освещение.
![ScreenShot167](https://user-images.githubusercontent.com/61763100/99708956-99a06080-2aaf-11eb-8232-b8c1eb172f91.png)

Все остальное тестируется, баг сложно воспроизвести.

**Comment by Daler-Karimov** [comment]
@Pospelove 
Пояснения к терминам бага:
1) "skybox" (скайбокс) - динамическая текстура неба зависящая от времени суток;
2) "алгоритм" - комлпекс действий для воспроизведения бага: зайти в локацию, выйти из игры, зайти в игру, выйти из локации.

##УСТАНОВЛЕНО
 Если внутри локации есть скайбокс, то при повторении алгоритма луна или солнце будет выглядеть нормально, но если в локации его нет, то при повторении алгоритма будет нарушена текстура луны или солнца на квадрат. Чтобы узнать есть ли скайбокс, надо пройти через стены локации и посмотреть на небо.

Примеры локаций где есть скайбокс: Храм Призывателей Ночи (00017081) - см. 1 и 2 скриншот, Смеющаяся крыса (00016A0E) - см. 3 скриншот.
![The_Elder_Scrolls_V_Skyrim_Special_Edition_Screenshot_2020 11 20_-_23 55 38 57](https://user-images.githubusercontent.com/61763100/99917130-62d08180-2d1f-11eb-8985-d5de0c2a7ed2.png)
![20201120235542_1](https://user-images.githubusercontent.com/61763100/99917131-6401ae80-2d1f-11eb-8efa-1b948401fe06.jpg)

Примеры локаций где нет скайбокса: Всякая всячина (00016A10) - см. 4 скриншот, Форт Сангард (000373ca) - см. 5 скриншот.
![ScreenShot95](https://user-images.githubusercontent.com/61763100/99917291-5dc00200-2d20-11eb-953a-cad02c579793.png)
![The_Elder_Scrolls_V_Skyrim_Special_Edition_Screenshot_2020 11 20_-_22 42 04 44](https://user-images.githubusercontent.com/61763100/99917294-631d4c80-2d20-11eb-839a-bf889c373a46.png)

**Comment by Daler-Karimov** [comment]
##УСТАНОВЛЕНО

При повторении алгоритма в дневное и вечернее время, перед камерой игрока появляется куб с фильтром освещения, скорее всего цвет фильтра зависит от времени суток. Ночью такого не происходит.

https://youtu.be/wRgKmoIaIsI
